### DISCUSSION TITLE
Update email domain for my school

### DISCUSSION THREAD

**Discussion by TheoLeCalvar** [community/community#110971] [GitHub Education]
### Hi everyone,

I work at [IMT Atlantique](https://www.imt-atlantique.fr/en) and I'd like to renew my teacher status but the system doesn't recognize my email as affiliated with the school.

Faculty member are issued an email with @imt-atlantique.fr domain whereas students have @imt-atlantique.net email. The system only recognize students emails and not mine.

Could you add the right domain for faculty members ?

**Comment by tammiramsey** [comment]
I don't have an answer for this, but I have the same issue. I put in a help ticket on January 2nd and still have NOT gotten a response, and I made a post about the same issue with my school domains on January 31. At this point I am so fed up that I may just move everything off the platform as far as my course assignments go, as I have gotten zero assistance and at this point it's getting ridiculous.
